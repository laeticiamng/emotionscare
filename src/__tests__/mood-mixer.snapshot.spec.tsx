import { describe, it, expect, vi } from "vitest";
import { render } from "@testing-library/react";
vi.mock("@/components/ui/PageHeader", () => ({
  default: ({ title }: any) => <div>{title}</div>,
}));

vi.mock("@/components/ui/button", () => ({
  Button: ({ children }: any) => <button>{children}</button>,
}));
import Page from "@/app/modules/mood-mixer/page";

describe("MoodMixer Page", () => {
  it("rend la page", () => {
    const { container } = render(<Page />);
    expect(container).toMatchSnapshot();
  });
});
