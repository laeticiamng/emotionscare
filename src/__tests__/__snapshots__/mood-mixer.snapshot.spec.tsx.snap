// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MoodMixer Page > rend la page 1`] = `
<div>
  <main
    class="p-4 space-y-4"
  >
    <div>
      MoodMixer
    </div>
    <p>
      Bienvenue dans le module MoodMixer.
    </p>
    <button>
      Commencer
    </button>
  </main>
</div>
`;
